tasks/composer.js

```diff
@@ -20,6 +20,16 @@ module.exports = function(grunt) {
         ],
       }
     });
+    grunt.config(['composer', 'update'], {
+      options: {
+        flags: [
+          'no-interaction',
+          'no-progress',
+          'prefer-dist'
+        ],
+      }
+    });
+    grunt.config(['composer', 'drupal-scaffold'], {});
```

Comment:
Yes, that is correct.  Since `grunt scaffold `requires the index.php and the drush status check, the `grunt composer:drupal-scaffold` needs to run before it.